README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 | Java / Groovy | Yes                | Yes                |
 | C++           | No                 | Yes                |
 | JavaScript    | No                 | Yes                |
-| gRPC          | -                  | Yes                |
```

Comment:
We don't offer a gRPC client. Clients use gRPC to talk to the server. I found it very confusing lumping gRPC amongst other clients, since every single client uses it. gRPC is used differently than our client APIs.

Reply:
You can easily communicate with the server using gRPC.